docs/development.rst

```diff
@@ -31,6 +31,10 @@ virtualenv is a command line application written in Python. To work on it, you'l
 - :pypi:`tox`: to automatically get the projects development dependencies and run the test suite. We recommend
   installing it using `pipx <https://pipxproject.github.io/pipx/>`_.
 
+.. note::
+
```

Comment:
I don't think this is the case and we already voted against adding os special installation. pipx should work

Reply:
My intention was to warn Debian users **not** to use the os specific installation but to use `pip` and `pipx` (or zipapp and other). I mean that with _alternative installation methods_.

I had installed the debian package `tox` (which installs `python3-virtualenv` as a dependency) and tried to invoke the development environment (`tox -e dev`) with the following traceback: 

```
.tox create: /home/user/repos/virtualenv/.tox/.tox
ERROR: invocation failed (exit code 1), logfile: /home/user/repos/virtualenv/.tox/.tox/log/.tox-0.log
================================================================================= log start ==================================================================================
ModuleNotFoundError: No module named 'virtualenv.activation.xonsh'

================================================================================== log end ===================================================================================
ERROR: InvocationError for command /usr/bin/python3 -m virtualenv --no-download --python /usr/bin/python3 .tox (exited with code 1)
``` 
Don't you think a warning would be helpful if other developers on Debian run into that issue?

Reply:
That like could be useful. But

Reply:
> That like could be useful. But

But? Would you suggest any changes?

Reply:
I meant not documentation but runtime error to be better 🥰

Reply:
Hm, but while this issue is still present (and I don't think I'll be able to fix it), wouldn't it be good to spread the information? 

Maybe someone will be inspired by the note to fix it!  😉

Reply:
They can get inspired by reading the bug tracker? 🙂

Reply:
Well, just trying to help (and doing some persuasive work 😉).

Many developers may run into that issue and not know what to do. If they find out what cause this has in the main development docs (which surely almost everyone will read), this'll bear no problem. I personally feel reading docs more comfortable than searching for old, stale (and potentially locked) issues.

Maybe we could add the note anyway **and** transer the problem to a new issue, which we could link to?

Reply:
I'd rather fix the underlying issue than patch the documentation like this 🤷‍♂️

Reply:
OK, I give up. Will open an issue.